### ISSUE TITLE
Update documentation for setTitleColor:forState:

### ISSUE THREAD

**Issue by CareerFairPlus** [hightower/HTHorizontalSelectionList#82]
`setTitleColor:forState:` works great when initializing the control, but if the color of the text needs to be updated at runtime without re-creating the control, then `reloadData` also needs to be called.  

The documentation should be updated to reflect this.

Thanks!

**Comment by erikackermann** [comment]
You are correct.  A PR with a change that you think makes this more clear would be much appreciated!
